examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -60,7 +60,8 @@ object SparkSQLExample {
 
   private def runBasicDataFrameExample(spark: SparkSession): Unit = {
     // $example on:create_df$
-    val df = spark.read.json("examples/src/main/resources/people.json")
+    val df = spark.read.json(
+      "file:///" + System.getProperty("user.dir") + "examples/src/main/resources/people.json")
```

Comment:
I just downloaded Spark 2.0.1 and executed `./bin/run-example sql.SparkSQLExample` in the spark dir. I am on a Mac (on which platform are you?). It only becomes an issue when you try to run the examples from a different location; then it will fail.